### ISSUE TITLE
Please allow recovering from 12-words seed phrases

### ISSUE THREAD

**Issue by cyanlink** [lightninglabs/lightning-app#1334]
Please allow recovering from 12-words seed phrases since many other mobile apps and wasabi wallet on desktop only generates 12-words seed.

**Comment by bolatovumar** [comment]
@cyanlink I don't think that would work since LND generates a 24-word seed phrase. See here: https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md#24-word-cipher-seeds
